integration/init/local-single-file-gogetter/expected/.ship/state.json

```diff
@@ -0,0 +1,9 @@
+{
+  "v1": {
+    "config": null,
+    "releaseName": "ship",
+    "upstream": "file::/Users/robert/go/src/github.com/replicatedhq/ship/integration/init/local-single-file-gogetter/input/ship.yaml",
```

Comment:
Not OK

Maybe change this to `<UPSTREAM>` and include the upstream as part of the string replacement? (the map[string]string{} on line 94 of integration/init/integration_test.go)

Reply:
oh yea this is why i didn't push this

Reply:
We do variable substitution for all the `init_app` tests: https://github.com/replicatedhq/ship/blob/master/integration/init_app/amazon-eks-template/expected/.ship/state.json
Seems like the idiom is `__upstream__` not `<UPSTREAM>` though